src/sentry/relay/config/__init__.py

```diff
@@ -515,8 +525,8 @@ def get_transaction_metrics_settings(
     """This function assumes that the corresponding feature flag has been checked.
     See _should_extract_transaction_metrics.
     """
-    metrics = []
-    custom_tags = []
+    metrics: List[Any] = []
+    custom_tags: List[Any] = []
```

Comment:
Can we give these the same types as https://github.com/getsentry/sentry/pull/41112/files#diff-5e30755537fcfd5687d4399272065e49affcc12a4cf7a0bc377cff5234b8a08aR502-R503?

Reply:
@jjbayer can you please point me to the correct link? This one seems broken :(

Reply:
`extractMetrics` and `extractCustomTags` in `TransactionMetricsSettings`.

Reply:
fixed 21964f58b5